doc/changelog.rst

```diff
@@ -4,6 +4,9 @@ Changelog
 Changes in Version 4.3
```

Comment:
Should we say `Changes in Version 4.3 (4.3.2)`?

Reply:
Done